app/enterprise/1.3-x/developer-portal/working-with-templates.md

```diff
@@ -302,3 +302,1181 @@ This is my second post!
+
+---
+
+- [Theme Members](#theme-members)
+  - [`theme.colors`](#usercolors)
+  - [`theme.color`](#usercolor)
+  - [`theme.fonts`](#userfonts)
+  - [`theme.font`](#userfont)
+
+---
+
+## Theme Members
+
+### `theme.colors`
+
+#### Description
+
+> Returns table of color vars and their values as key value pairs.
+
+#### Return Type
+
+```lua
+table
+```
+
+#### Usage
+
+##### theme.conf.yaml
+```yaml
+name: Kong
+colors:
+  primary:
+    value: '#FFFFFF'
+    description: 'Primary Color'
+  secondary:
+    value: '#000000'
+    description: 'Secondary Color'
+  tertiary:
+    value: '#1DBAC2'
+    description: 'Tertiary Color'
+```
+
+##### Template
+
+{% raw %}
+```lua
+{% for k,v in each(theme.colors) do %}
+  <p>{{k}}: {{v}}</p>
+{% end %}
+```
+{% endraw %}
+
+##### Output
+
+```html
+<p>primary: #FFFFFF</p>
+<p>secondary: #000000</p>
+<p>tertiary: #1DBAC2</p>
+```
+
+### `theme.color`
+
+#### Description
+
+> Takes color var by string argument, returns value.
+
+#### Return Type
+
+```lua
+string
+```
+
+#### Usage
+
+##### theme.conf.yaml
+```yaml
+name: Kong
+colors:
+  primary:
+    value: '#FFFFFF'
+    description: 'Primary Color'
+  secondary:
+    value: '#000000'
+    description: 'Secondary Color'
+  tertiary:
+    value: '#1DBAC2'
+    description: 'Tertiary Color'
+```
+
+##### Template
+
+{% raw %}
+```lua
+<p>primary: {{theme.color("primary")}}</p>
+<p>secondary: {{theme.color("secondary")}}</p>
+<p>tertiary: {{theme.color("tertiary")}}</p>
+```
+{% endraw %}
+
+##### Output
+
+```html
+<p>primary: #FFFFFF</p>
+<p>secondary: #000000</p>
+<p>tertiary: #1DBAC2</p>
+```
+
+
+### `theme.fonts`
+
+#### Description
+
+> Returns table of font vars and their values as key value pairs.
+
+#### Return Type
+
+```lua
+table
+```
+
+#### Usage
+
+##### theme.conf.yaml
+```yaml
+name: Kong
+fonts:
+  base: Roboto
+  code: Roboto Mono
+  headings: Lato
+```
+
+##### Template
+
+{% raw %}
+```lua
+{% for k,v in each(theme.fonts) do %}
+  <p>{{k}}: {{v}}</p>
+{% end %}
+```
+{% endraw %}
+
+##### Output
+
+```html
+<p>base: Roboto</p>
+<p>code: Roboto Mono</p>
+<p>headings: Lato</p>
+```
+
+### `theme.font`
+
+#### Description
+
+> Takes font name by string argument, returns value.
+
+#### Return Type
+
+```lua
+string
+```
+
+#### Usage
+
+##### theme.conf.yaml
+```yaml
+name: Kong
+fonts:
+  base: Roboto
+  code: Roboto Mono
+  headings: Lato
+```
+
+##### Template
+
+{% raw %}
+```lua
+<p>base: {{theme.font("base")}}</p>
+<p>code: {{theme.font("code")}}</p>
+<p>headings: {{theme.font("headings")}}</p>
+```
+{% endraw %}
+
+##### Output
+
+```html
+<p>base: #FFFFFF</p>
+<p>code: #000000</p>
+<p>headings: #1DBAC2</p>
+```
+
+## `str`
+
+#### Description
+
+> Table containing useful string helper methods
+
+#### Usage
+
+{% raw %}
+##### _.upper()_ example
+```lua
+<pre>{{ str.upper("dog") }}</pre>
+```
+{% endraw %}
+
+#### Methods
```

Comment:
![image](https://user-images.githubusercontent.com/3756245/82369361-80bdd200-99dc-11ea-8184-f65a3d145944.png)
These sections aren't rendering properly I don't think. Add a space between each per https://www.markdownguide.org/basic-syntax/#heading-best-practices and see if it looks like you want it too.

Reply:
I'm not sure what is best for this. Perhaps @Jlawlzz  has the answer

Reply:
It looks like we need a space between each one, happy to make the change!